python/ray/serve/config.py

```diff
@@ -84,6 +91,33 @@ def replicas_settings_valid(cls, max_replicas, values):
 
         return max_replicas
 
+    @validator("policy", always=True)
+    def serialize_policy(cls, policy, values):
+        """Serialize policy to be cloudpickled and base64 encoded utf8 string.
+
+        If policy is a string, it is possible to be already serialized when constructed
+        from protobuf. In this case, we just return the already serialized string. Else
+        we import the policy and serialize it before return.
+        """
+        if isinstance(policy, Callable):
+            imported_policy = policy
+        elif isinstance(policy, str):
+            try:
+                assert b64encode(b64decode(policy)).decode("utf-8") == policy
+                return policy
+            except binascii.Error:
+                imported_policy = import_attr(policy)
+            except AssertionError:
+                imported_policy = import_attr(policy)
+        else:
```

Comment:
When do we expect this `else` case to happen?

Reply:
good call! let me remove this!